Index: xserver-xorg-video-fbdev/man/fbdev.man
===================================================================
--- xserver-xorg-video-fbdev.orig/man/fbdev.man	2007-04-22 03:52:00.000000000 +0200
+++ xserver-xorg-video-fbdev/man/fbdev.man	2007-04-22 03:52:49.000000000 +0200
@@ -38,8 +38,8 @@
 driver and will use it if there are no video modes configured.
 .PP
 For PCI boards you might have to add a BusID line to the Device
-section.  See above for a sample line.  You can use \*q\__xservername__
--scanpci\*q
+section.  See above for a sample line.  You can use \*q__xservername__
+\-scanpci\*q
 to figure out the correct values.
 .PP
 The following driver
@@ -60,4 +60,4 @@
 __xservername__(__appmansuffix__), __xconfigfile__(__filemansuffix__), xorgconfig(__appmansuffix__), Xserver(__appmansuffix__),
 X(__miscmansuffix__), fbdevhw(__drivermansuffix__)
 .SH AUTHORS
-Authors include: Gerd Knorr, Michel Dänzer, Geert Uytterhoeven
+Authors include: Gerd Knorr, Michel D\(:anzer, Geert Uytterhoeven
