Index: xserver-xorg-video-fbdev/src/fbdev.c
===================================================================
--- xserver-xorg-video-fbdev.orig/src/fbdev.c
+++ xserver-xorg-video-fbdev/src/fbdev.c
@@ -670,6 +670,12 @@
 	       pScrn->offset.red,pScrn->offset.green,pScrn->offset.blue);
 #endif
 
+#if GET_ABI_MAJOR(ABI_VIDEODRV_VERSION) >= 9
+	pScreen->canDoBGNoneRoot = TRUE;
+#else
+	pScrn->canDoBGNoneRoot = TRUE;
+#endif
+
 	if (NULL == (fPtr->fbmem = fbdevHWMapVidmem(pScrn))) {
 	        xf86DrvMsg(scrnIndex,X_ERROR,"mapping of video memory"
 			   " failed\n");
