Index: xserver-xorg-video-fbdev/src/fbdev.c
===================================================================
--- xserver-xorg-video-fbdev.orig/src/fbdev.c	2011-01-28 17:52:29.510667250 +1100
+++ xserver-xorg-video-fbdev/src/fbdev.c	2011-01-28 17:53:59.150667250 +1100
@@ -404,6 +404,12 @@
 
 	pScrn->monitor = pScrn->confScreen->monitor;
 
+#if GET_ABI_MAJOR(ABI_VIDEODRV_VERSION) >= 9
+	pScrn->pScreen->canDoBGNoneRoot = TRUE;
+#else
+	pScrn->canDoBGNoneRoot = TRUE;
+#endif
+
 	FBDevGetRec(pScrn);
 	fPtr = FBDEVPTR(pScrn);
 
